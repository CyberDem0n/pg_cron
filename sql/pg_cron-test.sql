CREATE EXTENSION pg_cron VERSION '1.0';
SELECT extversion FROM pg_extension WHERE extname='pg_cron';
ALTER EXTENSION pg_cron UPDATE TO '1.3';
SELECT extversion FROM pg_extension WHERE extname='pg_cron';

-- Vacuum every day at 10:00am (GMT)
SELECT cron.schedule('0 10 * * *', 'VACUUM');

-- Stop scheduling a job
SELECT cron.unschedule(1);

-- Vacuum every day at 10:00am (GMT)
SELECT cron.schedule('vacuum', '0 10 * * *', 'VACUUM');

-- Stop scheduling a job
SELECT cron.unschedule('vacuum');

DROP EXTENSION pg_cron;
