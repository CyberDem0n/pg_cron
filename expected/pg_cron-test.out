CREATE EXTENSION pg_cron VERSION '1.0';
SELECT extversion FROM pg_extension WHERE extname='pg_cron';
 extversion 
------------
 1.0
(1 row)

ALTER EXTENSION pg_cron UPDATE TO '1.3';
SELECT extversion FROM pg_extension WHERE extname='pg_cron';
 extversion 
------------
 1.3
(1 row)

-- Vacuum every day at 10:00am (GMT)
SELECT cron.schedule('0 10 * * *', 'VACUUM');
 schedule 
----------
        1
(1 row)

-- Stop scheduling a job
SELECT cron.unschedule(1);
 unschedule 
------------
 t
(1 row)

-- Vacuum every day at 10:00am (GMT)
SELECT cron.schedule('vacuum', '0 10 * * *', 'VACUUM');
 schedule 
----------
        2
(1 row)

-- Stop scheduling a job
SELECT cron.unschedule('vacuum');
 unschedule 
------------
 t
(1 row)

DROP EXTENSION pg_cron;
